tools/cheribsdbox/Makefile

```diff
@@ -377,7 +377,6 @@ CRUNCH_ALIAS_nmtree= mtree
 # It seems like ${LIBNETBSD} works:
 CRUNCH_LIBS+=	${LIBNETBSD}
 
-# usr/sbin/de4ctl
 # usr/sbin/flashit
```

Comment:
flashit is also de4 specific and can go

Reply:
I axed it along with some other nearby lines that looked DE4-specific.